simple.hs

```diff
@@ -0,0 +1,15 @@
+module Main where
+
+import Data.Char (toLower)
```

Comment:
Why not use `Data.Text`?

Reply:
The README said that only the stdlib is allowed. Data.Text isn't in base right? Otherwise I would have used it (or ByteString).

Reply:
Ok, didn't read the readme until that point. =)
`text` is sort of an stdlib because it is distributed with ghc the same way `base` is. But it sort of isn't because its version may change independently from ghc. So I'm not sure now if it's allowed.